.github/workflows/ci.yml

```diff
@@ -226,17 +226,31 @@ jobs:
           iscc installers/installer.iss
           mv installers/Output/setupSasView.exe installers/dist
 
-      - name: Build sasview installer dmg file (OSX)
+      - name: Sign executable and create dmg (OSX)
```

Comment:
Do we need a conditional here to only sign when pushed to main? I only ask because this is diverging from our previous behavior.

Reply:
Yes, it will make sense. From the user's perspective, signing only makes sense when it is followed by notarization. (Gatekeeper complains when app signed but not notarized).

Reply:
And we do notarization from nightly-build on main branch

Reply:
If I understand correctly, as it is now:
```
 pull_request:
 push:
    branches: [main]
    tags: ['v*']
```
is done for pushes to main and all PRs, which is probably something we actually want?